src/ProgressOnderwijsUtils/ProgressOnderwijsUtils.csproj

```diff
@@ -1,8 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\NugetPackagesCommon.props" />
   <PropertyGroup Label="Configuration">
-    <Version>91.2.1</Version>
-    <PackageReleaseNotes>Make CssClass as parameter nullable.</PackageReleaseNotes>
+    <Version>91.2.2</Version>
+    <PackageReleaseNotes>Change _class(string) method used in tests to _class(CssClass).</PackageReleaseNotes>
```

Comment:
Testen worden niet meegepubliceerd, dus hiervoor hoef je niet te bumpen.

Echter het weghalen van _class met de params-strings zou wel een nieuwe versie kunnen worden.

Persoonlijk denk ik niet dat dat nu hoeft, dus wmb doen we geen version bump, maar als je dat wel al wil, dan gaat het over de API wijzigingen in de utils-zelf, niet de testen.

Reply:
Oke goed om te weten!